arbnode/transaction_streamer.go

```diff
@@ -196,6 +196,35 @@ func deleteStartingAt(db ethdb.Database, batch ethdb.Batch, prefix []byte, minKe
 	return iter.Error()
 }
 
+// deleteFromRange deletes key ranging from startMinKey(inclusive) to endMinKey(exclusive)
+func deleteFromRange(db ethdb.Database, prefix []byte, startMinKey uint64, endMinKey uint64) error {
```

Comment:
Is there a better place for this function? perhaps arbutil or somewhere else?
Also - I think that it can be used in some places (around reorg) but not sure.

Reply:
Not sure, I mainly placed it here cause deleteStartingAt is also present here, also moving to some other package will require dbKey and uint64ToKey as well to that package.

But let me know if you feel it should be moved, I can move dbKey and uint64ToKey out as well.